### ISSUE TITLE
why sorting matters?

### ISSUE THREAD

**Issue by eveliao** [lancopku/SGM#7]
In the paper, you mentioned that "In addition, the proposed models are trained using the maximum likelihood estimation method and the cross-entropy loss function, which requires humans to predefine the order of the output labels. "
but this doesn't explain for me... It means that order matters because we use seq2seq model(instead of the mentioned 'MLE and cross-entropy'), which is the intrinsic property of model. The prediction order should follow the true order. Thus human-defined order has impact.
Is the reason that high-frequency labels are more likely the root label in a hierarchical category taxonomy and thus more likely to have correlations with other labels?

**Comment by ypengc7512** [comment]
Yeah, for the Seq2Seq model we need to pre-define the order of the output labels. For the label order problem, you can refer to our latest work: https://arxiv.org/pdf/1809.03118.pdf

**Comment by Imorton-zd** [comment]
@ypengc7512 Hi, would you please open the source code of the latest work (seq2set), as you mentioned above. Many thanks.
